docs/stale-subvolume.md

```diff
@@ -0,0 +1,38 @@
+# subvolume cleanup
```

Comment:
Rather than having a specific title of `subvolume cleanup`, IMO, we have have more generic title like `subvolume`.
Currently it will support `listing` and `cleaning` of stale volumes.  Later on more subvolume related commands can be added to this doc if needed.